content/applications/finance/accounting/getting_started/cheat_sheet.rst

```diff
@@ -0,0 +1,302 @@
+:code-column:
+:custom-css: accounting.css
+:custom-js: accounts.js,chart-of-accounts.js,entries.js,misc.js,reconciliation.js
+
+======================
+Accounting cheat sheet
+======================
+
+.. h:div:: intro-list
+
+   .. rst-class:: intro-balance
+
+   The **Balance Sheet** is a snapshot of the company's finances at a specific date (as opposed to
+   the Profit and Loss, which is an analysis over a period);
+
+   * .. rst-class:: intro-assets
+
+     **Assets** represent the company's wealth, things it owns. Fixed assets includes buildings and
+     offices, and current assets include bank accounts and cash. A client owing money is an asset.
+     An employee is not an asset.
+
+   * .. rst-class:: intro-liabilities
+
+     **Liabilities** are obligations from past events that the company will have to pay in the
+     future (utility bills, debts, unpaid suppliers). Liabilities could also be defined as a source
+     of financing which is provided to the company, also called *leverage*.
```

Comment:
updated. I also replaced the second "and" by "while" to avoid repetition: 
Fixed assets include buildings and offices, while current assets include bank accounts and cash.